### PR TITLE
Endpoint tests

### PR DESCRIPTION
**Pull request by mikefab**
Add tests for api endpoints: travel_from_country_activity

### CODE REVIEW

File: app/helpers/amadeus.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ */
+function form_mobility(collection, obj, blob_name) {
+  var mobility = {};
+  if (obj.year) {
+    // obj.year is simply a year: 2016
+    var year_date = moment(obj.year + '-01-01');
+    mobility.date = new Date(
+      moment(year_date).add(obj.week, 'weeks').format("YYYY-MM-DD")
+    ).toISOString();
+    mobility.date_to = new Date(
```
Comment (qjouda) [inline app/helpers/amadeus.js]: I think moment support toISOString function
```js
moment(year_date).add(obj.week, 'weeks').toISOString()
```

File: lib/import/amadeus_mobility.js [hunk 2]
```diff
@@ -4,17 +4,12 @@ var config = require('../../config');
 var async = require('async');
 var bluebird = require('bluebird');
 var util = require('../../util');
-var Mobility = require('../../app/models/mobility');
-var moment = require('moment');
-var csv = require('fast-csv');
+var helper = require('../../app/helpers/amadeus');
```
Comment (qjouda) [inline lib/import/amadeus_mobility.js]: 👍

File: server.js [hunk 3]
```diff
@@ -95,19 +95,19 @@ router.route('/mobility_populations/:country_code/:start_time?/:end_time?')
...
-      p.end_date
+      p.end_date,
+      p.country_iso
     ).then(res.json.bind(res)).catch(next);
   });
 
 router.route('/summary_azure/:container')
   .get(apicache('1 day'), function(req, res, next) {
-    util.summary_azure(req.params.container).then(res.json.bind(res)).catch(next);
+    helper.summary_azure(req.params.container).then(res.json.bind(res)).catch(next);
```
Comment (qjouda) [inline server.js]: just wondering why 'next' is being called on 'catch'?

File: app/helpers/amadeus.js (hunk 1 above)
Comment (mikefab) [inline app/helpers/amadeus.js]: .format("YYYY-MM-DD") removes UTC offset: 2016-09-23T00:00:00.000Z
versus
moment(year_date).add(obj.week, 'weeks').toISOString() : 2016-09-23T04:00:00.000Z

File: server.js (hunk 3 above)
Comment (mikefab) [inline server.js]: Good point. We'll do this as a separate task.

**Pull request merged**